src/utils/command-helpers.ts

```diff
@@ -300,3 +301,14 @@ export const nonNullable = <T>(value: T): value is NonNullable<T> => value !== n
 export const noOp = () => {
   // no-op
 }
+
+export const checkFileForLine = (filename: string, line: string) => {
+  let filecontent = ''
+  try {
+    filecontent = fs.readFileSync(filename, 'utf8')
+  } catch (error_) {
+    // @ts-expect-error TS(2571) FIXME: Object is of type 'unknown'.
```

Comment:
Pull in the main branch to get updates from this PR https://github.com/netlify/cli/pull/6877 to fix this type error